addons/point_of_sale/models/pos_order.py

```diff
@@ -558,6 +558,18 @@ def refund(self):
             'target': 'current',
         }
 
+    def action_receipt_to_customer(self, receipt, cashier_id, client):
+        template_obj = self.env['mail.mail']
+        message = "<p>Dear %s,<br/>Here is your electronic ticket from the %s. </p>" % (client['name'], self.id)
+        template_data = {
+            'subject': 'Receipt %s' % self.id,
+            'body_html': message + '<img src="data:image/jpeg;base64,%s"/>' % receipt,
+            'email_from': self.env['res.users'].search([('id', '=', cashier_id)]).email,
```

Comment:
check with laura, but we probably don't want to use the email of the user, but the mail of the admin or something like this

Reply:
In most case, the email of the current company is the more relevant. Something like contact@mycompany.com
Admin email will be wrong, specially in a multi company context.